include/pagmo/population.hpp

```diff
@@ -79,7 +79,7 @@ class population
      * Constructs an empty population with a pagmo::null_problem.
```

Comment:
I am presuming the champion machinery will be documented in the class' description eventually?